src/styles/Base/variables.scss

```diff
@@ -5,6 +5,14 @@ $font-links: 'Montserrat', sans-serif;
 /* Colors */
 
 $black: #1F1C17;
+$brown: #c09569;
+$lighter-black: #1d1d1d;
```

Comment:
I propose a different system of naming. Google material design names colors by numbers so:
Say our main black is `$black-500: #1F1C17;`
We can do this:
`$black-400`
`$black-300`
`$black-200`
`$black-100`

This says shades of black that are lighter by degree from the main color. So the main color has a level of 500 a less darker color will have a level of 400 and even a less darker one a level of 300 you get the point. This way we can manage shades better.
This will also work for darker shades so:
`$black-600`
`$black-700`
will be darker shades of our "black"

This practice will save us from doing something like this:
`light-black`
`lighter-black`
`even-lighter-black`
`lightest-black`
etc. You get the point. This easily becomes unmanagable